article/sections/_2_graph_importance.tex

```diff
@@ -176,25 +189,37 @@ \section{Perils of disregard}
 For our purposes, we focus on car-centric modes (Drive alone, Shared ride with another person,
 and a shared ride with two or more individuals).
 We compute the difference in mode choice probabilities from different models based on the two constructed DAGs.
+Readers interested in exploring the details of our simulation exercise can refer to our GitHub repository \citep{brathwaite_etal_2020}.
 
 
 
 We then plot histograms of the computed differences between the average probability of an individual
 in our sample choosing a car centric mode before and after implementing a policy or intervention
 aimed at reducing travel distance.
-These differences are plotted under the two different assumptions about the data generating process illustrated in the causal graphs above.
-Figure \ref{fig:histogram_probability} highlights the bias between the estimated probability of an average individual choosing a car centric mode.
-This difference shows the importance of considering the data generating process when estimating transportation demand models aiming to forecast the impact of proposed policies.
+These differences are plotted under the following assumptions: that the outcome model is the same in both scenarios and that the only difference between the two scenarios is the set of variables affected by the proposed intervention.
+This difference in the set of variables affected by the proposed intervention is a result of the two different causal graphs representing each scenario (Figures \ref{fig:IND_GRAPH} and \ref{fig:DA_causal_2}).
+Figure \ref{fig:histogram_probability} highlights the resulting bias between the estimated probability of an average individual choosing a car centric mode.
+This plotted difference shows the importance of using causal graphs in the estimation of treatment effects.
+Of course, the importance of a correct causal graph does not and should not take away from the importance of correctly specified outcome choice models.
 
-\begin{figure}
+\begin{figure}[h!]
    \centering
    \includegraphics[width=0.5\textwidth]{histogram_selection_on_obs.png}
    \caption{Histograms of the probability of choosing Car Centric Modes under Different Data Generating processes.}
    \label{fig:histogram_probability}
 \end{figure}
 
-The data generating process might not be easily distinguishable in the majority of situations, mainly due to the complexity of the real world.
+We have shown in this simulation exercise how causal graphs can be helpful in avoiding biased causal effect inferences of proposed treatments or policies.
+Specifically, two situations highlight this fact; having mediating or confounding variables relative to the variable the proposed treatment intervenes on in our outcome model.
+In the case of mediating variables relative to the node the treatment intervenes on, we need to be careful about constructing our treatment effect estimator and make sure we have a total effect estimator instead of a natural direct effect estimator.
```

Comment:
```suggestion
In the case of mediating variables relative to the node the treatment intervenes on, we need to be careful about constructing our treatment effect estimator.
In particular, we need to ensure that we appropriately use total effect estimators instead of natural direct effect estimators.
```